apps/web/src/lang/fr.json

```
@@ -6585,6 +6613,10 @@
     "defaultMessage": "Créer une classification",
     "description": "Page title for the classification creation page"
   },
+  "faS0+E": {
+    "defaultMessage": "Traiter l’état",
+    "description": "Title for card for actions related to changing a process' status"
```

Don't forget the id on the English string!
```suggestion
    "description": "Title for card for actions related to changing the status of a process"
```